tests/reporter/kubernetes.go

```diff
@@ -1070,7 +1070,12 @@ func getVmiType(vmi v12.VirtualMachineInstance) (string, error) {
 		}
 	}
 
-	return "", fmt.Errorf("unknown type, vmi %s", vmi.ObjectMeta.Name)
+	vmiJson, err := json.Marshal(vmi)
+	if err == nil {
```

Comment:
what about `if vmiJson, err := json.Marshal(vmi); err == nil {` ?

Reply:
right, let's be go-idiomatic.